root_s3_object_lifecycle.tf

```diff
@@ -5,6 +5,9 @@ locals {
   backend_checks_bucket_policy_status = "Enabled"
   dirty_bucket_policy_status          = local.environment == "prod" ? "Disabled" : "Enabled"
 
+  object_tag_dr2_ingest_key   = "PreserveDigitalAssetIngest"
+  object_tag_dr2_ingest_value = "Complete"
```

Comment:
Rename this to: `object_tag_dr2_ingest_value_complete`

Makes it easier to understand in the places where the local is used.